tests/common/test_core_api.py

```diff
@@ -1,5 +1,6 @@
 import pytest
-from cow_py.common import constants, cow_error, chains, config
+
+from cow_py.common import chains, config, constants, cow_error
 
 
 @pytest.mark.parametrize("module", [constants, cow_error, chains, config])
```

Comment:
I would like to see a real integration test (without mocked HTTP requests) which touch cowswap API, place an unrealistic order (e.g. buy ETH for 1 USD), cancel it and axit.
I apologize if such a test already exists (in that case, please close this comment), but I couldn't find it in the codebase from this PR.

Reply:
On this PR, the API module didn't exist yet. This PR is just creating the codegen and adding some common utils. All API related logic and tests should be integrated here: https://github.com/cowdao-grants/cow-py/pull/13

Reply:
This will be done on another PR
https://github.com/cowdao-grants/cow-py/pull/12/files#r1744309895
Closing this